dspace/config/spring/api/external-services.xml

```diff
@@ -92,5 +94,38 @@
         </property>
     </bean>
 
+    <bean id="scopusLiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="ScopusImportService"/>
+        <property name="sourceIdentifier" value="scopus"/>
+        <property name="recordIdMetadata" value="dc.identifier.scopus"/>
+        <property name="supportedEntityTypes">
+            <list>
+                <value>Publication</value>
```

Comment:
Should this also support "none" (i.e. normal Items)?

Reply:
Feedback wasn't addressed. Can this support "none" type (i.e. regular items)?  The same question goes for all these import sources.